ios/INVRegionMonitor.m

```diff
@@ -138,13 +138,19 @@ - (BOOL)_isRegionMonitoringPossible:(CLAuthorizationStatus)status {
     );
 }
 
-- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit {
+- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit locationManager:(CLLocationManager *)locationManager {
     NSDictionary *body = @{
         @"region": @{
             @"identifier": identifier,
         },
         @"didEnter": @(didEnter),
         @"didExit": @(didExit),
+        @"accuracy": @(locationManager.location != nil ? locationManager.location.horizontalAccuracy : 99999),
+        @"bearing": @(locationManager.location != nil ? locationManager.location.course : 0),
```

Comment:
You can find the iOS docs of the location object at  https://developer.apple.com/documentation/corelocation/cllocation.